crates/typst/src/layout/measure.rs

```diff
@@ -48,6 +49,18 @@ pub fn measure(
     context: Tracked<Context>,
     /// The callsite span.
     span: Span,
+    /// The width available to layout the content.
+    ///
```

Comment:
Maybe we should have an example showcasing the difference here?

Reply:
I’ve added an example. How does it look?